lib/lnc.ts

```diff
@@ -78,6 +78,38 @@ export default class LNC {
         );
     }
 
+    get status() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientStatus &&
+            this.wasm.wasmClientStatus()
+        );
+    }
+
+    get expiry() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientGetExpiry &&
+            new Date(this.wasm.wasmClientGetExpiry() * 1000).toLocaleString()
```

Comment:
I don't think it's a good idea to return a string for the date because users may want to compare this value to other dates. It would be better to return the `Date` or a `number`